SETUP.md

```diff
@@ -66,6 +95,17 @@ To perform backup, BBQ needs rights to read BigQuery data from the project which
     ```
 * (Optionally) Configure schedule time and kinds to export in [cron.yaml](./config/cron.yaml) file.
 
+### Security Layers
+
+ * **Firewall**
+   * A firewall provides identity-agnostic access control for your App Engine app.
+   Current firewall setup only allows GAE cron requests and GAE task queue requests.
+ * **IAP**
+   * Cloud [Identity-Aware Proxy](https://cloud.google.com/iap/docs/concepts-overview) (Cloud IAP) lets you manage access to GAE application.
+     Standard GAE application uses primitive roles, so it gives wider access than necessary,
```

Comment:
I'm not sure if I understand this. We say that GAE uses default service account with Editor role but this is all within GCP. As far as I understand IAP is about external access to GAE services.

Reply:
It was more like a comparison of access level, maybe you have some propositions about how to rephrase this.
GAE roles - anyone from the web can access, the only limitations are roles (potentially wide group of people can get through)
IAP - control traffic more restrictive (only whitelisted accounts will have access)

Reply:
I'll remove this sentence